src/Orc.CsvTextEditor/Extensions/TextEditorExtensions.cs

```diff
@@ -71,6 +71,8 @@ public static List<ICompletionData> GetCompletionDataForText(this TextEditor tex
                 var columnChunk = text.Substring(columnOffset, columnWidth);
                 var words = columnChunk.Split(null);
                 var currentWord = text.GetWordFromOffset(offset - 1);
+
+                autocompletionText = autocompletionText.ToLower();
```

Comment:
we have another loop. The loop of lines and probably we have to move `autocompletionText.ToLower()` out of this loop either